README.md

```diff
@@ -50,6 +50,9 @@ check.addEventListener('error', function(event) {
 const input = check.querySelector('input')
 
 input.addEventListener('auto-check-send', function(event) {
+  // `auto-check-send` is cancellable
+  event.preventDefault()
```

Comment:
The addition of `preventDefault` here sabotages the rest of the code in this usage example (the one that adds `custom_form_data` to the form body)